theano/gpuarray/opt.py

```diff
@@ -519,10 +517,10 @@ def local_gpuflatten(node, context_name, inputs):
 @register_opt('fast_compile')
 @op_lifter([tensor.Elemwise])
 @register_opt2([tensor.Elemwise], 'fast_compile')
-def local_gpu_elemwise(node, context_name, inputs):
-    op = node.op
+def local_gpu_elemwise(op, context_name, inputs):
     scal_op = op.scalar_op
     name = op.name
+    node = op.make_node(*inputs)
```

Comment:
I understand I'm creating what has been removed for the good. But, I do not understand the condition in l:527